src/main/java/org/humancellatlas/ingest/security/DefaultAccountService.java

```diff
@@ -0,0 +1,25 @@
+package org.humancellatlas.ingest.security;
+
+import org.humancellatlas.ingest.security.exception.DuplicateAccount;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DefaultAccountService implements AccountService {
+
+    private final AccountRepository accountRepository;
+
+    public DefaultAccountService(AccountRepository accountRepository) {
+        this.accountRepository = accountRepository;
+    }
+
+    @Override
+    public Account register(Account account) {
+        account.addRole(Role.CONTRIBUTOR);
```

Comment:
If the `account` is a GuestAccount instance, won't it have the Guest role? We should probably remove that role here too.

Alternatively, we should change the model a bit here to reduce confusion. register() should be solely responsible for creating Accounts, and so it shouldn't take an Account as input but rather information for creating an Account.